addons/hr_holidays/static/src/owl/components/thread/thread.js

```diff
@@ -0,0 +1,51 @@
+odoo.define('hr_holidays.component.Thread', function (require) {
+'use strict';
+
+var Thread = require('mail.component.Thread');
+
+var core = require('web.core');
```

Comment:
```js
const { _t } = require('web.core');
```

(as a side note, even though it looks a bit like "magic" when we are not that used to ES6, this syntax will help use migrate code when we will use native JS modules, e.g. with `import { _t } from "web.core"`)

Reply:
ok